constellations/orion/src/tests_integration/class_integration_test.go

```diff
@@ -195,6 +195,7 @@ func createClass(id int) domains.Class {
 			StartDate:       now,
 			EndDate:         later1,
 			GoogleClassCode: domains.NewNullString("ab12cd34"),
+			PaymentNotes:    domains.NewNullString("notes1"),
```

Comment:
Can you test Null notes somewhere in these tests? Doesn't have to be here.

Reply:
I meant test "" string for `notes`.

Reply:
oh i did, i guess i just didn't push it that way...